docs/multisig-guide.md

```diff
@@ -0,0 +1,115 @@
+
+## What is Single-sig?
+
+You are probably familiar with using 1 *signing device* (also called *hardware wallet*) that generated 1 seed. 
+
+Pros:
+
+* 1 seed is all you need to access your funds
+* 1 seed is simple
+* Only 1 secure location to store the seed backup is needed
+
+Cons:
+
+* If you get tricked into giving hackers your seed your Bitcoin will be stolen immediately (see Phishing attacks [here](https://www.youtube.com/watch?v=B-09WDPXZmU)).    1 mistake on your part and the Bitcoin are lost.
+
+## What is Multisig?
+
+Multisig is a way to allow spending from your wallet only after 2 *signing devices*  (e.g. 2 *hardware wallets*) have signed the outgoing transaction.  These 2 *signing devices* can be any 2 out of 3 *signing devices*. This is called a *2 of 3* multisig wallet.
+
+Pros:
+
+* Way more secure! 1 mistake (e.g. 1 seed lost or stolen) cannot result in loss of funds. 
+  * Useful for long term storage of significant funds
+  * If 1 seed was lost or stolen, all the funds can be transferred to a new setup with the 2 remaining seeds + all *(master) public keys* 
+
+Cons:
+
+* More complicated
+* 3 secure locations to store the seed backups are needed
+* *(master) public keys* backup needed
+
+#### Further information
+
+* [https://bitcoiner.guide/multisig/intro/#why-multisig](https://bitcoiner.guide/multisig/intro/#why-multisig)
+
+* [https://btcguide.github.io/why-multisig](https://btcguide.github.io/why-multisig)
+* [https://coinguides.org/multisig-multi-signature-wallet/](https://coinguides.org/multisig-multi-signature-wallet/)
+
+
+
+## Overview
+
+### 1. Seed creation
+
+* You need 3 *signing devices* (see [here](multisig-security-tradeoffs.md) for a comparison, ideally the 3 devices are from 3 different vendors), that generate 3 different seeds.
+* The 3 seeds need to be stored on steel to survive long term, and resistant to fire & water
+
+You should have now:
+
+![secrets](images/multisig-guide/secrets.svg)
+
+#### Seed storage locations
+
+This is a crucial part of the setup and needs **careful consideration**. Only if you choose these locations well you get the benefits (see [https://bitcoiner.guide/multisig/intro/#multisig-benefits](https://bitcoiner.guide/multisig/intro/#multisig-benefits)). Here a few points to pay attention to:
+
+* Seed backups (on steel) are not protected by a pin as the *signing device*s (e.g. *hardware wallets*). They therefore need a **secure location**, i.e. accessible only to you or people you completely trust.
+  * Examples are: Second home/apartment, Safe deposit box, a relatives (or very good friends) house/apartment
+* In 1 location there should not be more than 1 seed
+  * Minimum is: 3 **secure locations**, in each location is the signing device and its seed backup
+  * Best is: 3 **secure locations** for the seed backups **and** 3 (medium) secure locations for the *signing device*s (e.g. *hardware wallets*).
```

Comment:
I would add here, that ephemeral devices might reduce the number of backups you need. Also an emergency key might not even need a device. So e.g. one emergency key without a device and one ephemeral device reduces the number of secure locations to 4.

Reply:
Ok, I added a bullet point addressing this: https://github.com/cryptoadvance/specter-desktop/commit/c58a4bec0140f5278d62063fd94bc4e8072a6d83